mgmt/api/CoreAPI.h

```diff
@@ -82,6 +82,6 @@ TSMgmtError EventIsActive(const char *event_name, bool *is_current);
 TSMgmtError EventSignalCbRegister(const char *event_name, TSEventSignalFunc func, void *data);
 TSMgmtError EventSignalCbUnregister(const char *event_name, TSEventSignalFunc func);
 
-TSMgmtError HostStatusSetDown(const char *name);
+TSMgmtError HostStatusSetDown(const char *name, const int down_time);
```

Comment:
I never do "const" on an int like this, which is call by value. It's harmless, but can be confusing.